### DISCUSSION TITLE
log from docker image

### DISCUSSION THREAD

**Discussion by magimat** [aqualinkd/AqualinkD#178] [General]
Trying to run aqualinkd inside a docker container.

I tried your other repo with a beta dockerfile, seems to work, but when I run it, I don't get any log output from then container... any idea?

**Comment by sfeakes** [comment]
What are you using to run / start the container?   Using the instructions in the aqualinkd-docker repo, you should get the output of aqualinkd written to the console.

If you want the log written to a local file (outside the container) then add the following to docker-compose.yml under the `volumes:` section
```
- type: bind
        source: ./aqualinkd.log
        target: /aquacfg/aqualinkd.log
        read_only: false
```
And change your aqualinkd.conf to point to that mount for logging
```
log_file=/aquacfg/aqualinkd.log
```

**Comment by magimat123** [comment]
turns out my problem was that my config file wasn't at the right place, everything works now.

thanks!
